app/src/main/res/values/strings_settings.xml

```diff
@@ -2,18 +2,18 @@
 <resources>
 
     <!-- Settings interface -->
-    <string name="title_settings">Settings / Help</string>
+    <string name="title_settings">Settings and help</string>
     <string name="title_customization">Customization</string>
     <string name="title_appearance_settings">Appearance</string>
     <string name="title_appearance_settings_general">General appearance</string>
-    <string name="title_appearance_settings_applications">Applications appearance</string>
+    <string name="title_appearance_settings_applications">Appearance of apps</string>
     <string name="title_behavior_settings">Operation</string>
-    <string name="title_help_and_general_info">Help and general information</string>
+    <string name="title_help_and_general_info">Help and general info</string>
 
     <!-- Notification -->
-    <string name="set_notification">Notification for favorites access</string>
+    <string name="set_notification">Notification for access to favorites</string>
     <string name="set_notification_help">Uses a transparent icon</string>
-    <string name="notification_title">Access to favorites</string>
+    <string name="notification_title">Favorites</string>

```

Comment:
It is still somewhat non-intuitive that the same string (?) is used for the title field, and that clicking it does anything.
I think "🔧 manage favorites" in the title field of the notification would help.

Reply:
Currently it is indeed the same string which is used for both, but this can easily be changed.
If we keep the "Manage favorites" from strings.xml we could reuse this string for the title of the notification (for me it would make sense because both the menu entry and the notification title are leading to the same interface).

Reply:
Got that out of the way filed as https://github.com/falzonv/discreet-launcher/issues/85